src/answers-umd.js

```diff
@@ -514,14 +514,29 @@ class Answers {
   }
 
   /**
-   * Processes a translation which includes performing interpolation, pluralization, or both
-   * @param {string} translations The translations, or a stringified JSON of possible translations
+   * Processes a translation which includes performing interpolation, pluralization, or
+   * both
+   * @param {string | Object} translations The translation, or an object containing
+   * translated plural forms
    * @param {Object} interpolationParams Params to use during interpolation
    * @param {number} count The count associated with the pluralization
+   * @param {string} language The langauge associated with the pluralization
    * @returns {string} The translation with any interpolation or pluralization applied
    */
-  processTranslation (translations, interpolationParams, count) {
-    return TranslationProcessor.process(translations, interpolationParams, count);
+  processTranslation (translations, interpolationParams, count, language) {
+    const initLocale = this._getInitLocale();
+    language = language || initLocale.substring(0, 2);
+
+    return TranslationProcessor.process(translations, interpolationParams, count, language);
+  }
+
+  /**
+   * Gets the locale that ANSWERS was initialized to. Defaults to 'en'
+   *
+   * @returns {string}
+   */
+  _getInitLocale () {
+    return this.core.globalStorage.getState(StorageKeys.LOCALE) || 'en';
```

Comment:
I think we already have some logic in this class that defaults the locale to `en`. We should make sure we're not applying defaults in two places.

Reply:
Ah, that's right. Removing this.